personal_assistant/personal_assistant/settings.py

```diff
@@ -45,6 +45,7 @@
     'django.contrib.staticfiles',
     'pa_core',
     'pa_user',
+    'contacts', #Додав свій застосунок contacts 
```

Comment:
Згідно 1-го пункту [звідси](https://python24softw-r7k7319.slack.com/archives/C07PM6YRGBZ/p1728048391294319) потрібно переіменувати застосунок (звісно і в усіх інших місцях де зустрічається ця назва) і підправити коментар згідно стандартів наступним чином:
```suggestion
    'pa_contact',  # Додав застосунок для контактів
```
Також пізніше не буде зрозуміло по коментарю чий це свій тож можна трішки переформулювати.